components/ELink.tsx

```diff
@@ -2,7 +2,7 @@ import Link from 'next/link';
 import React from 'react';
 
 export interface ELinkProps {
-  link: string;
+  link: string | null;
```

Comment:
Hm, I'm not sure if this is the behaviour that makes sense for this component. I think the caller should handle whether or not the `link` is null; that way, we can keep this component's functionality the same (an external link).